package.json

```diff
@@ -67,6 +67,7 @@
     "postcss-loader": "0.9.1",
     "prop-types": "^15.5.10",
     "raw-loader": "0.5.1",
+    "smoothscroll-polyfill": "^0.3.5",
```

Comment:
@markdalgleish we are locking other packages to explicit versions, should the `^` be removed here?